apple/Sources/Sargon/Util/SargonObjectCodable.swift

```diff
@@ -8,16 +8,14 @@ public protocol SargonObjectCodable: Codable {
 
 extension SargonObjectCodable {
 	public func encode(to encoder: any Encoder) throws {
-		let dict = try JSONSerialization.jsonObject(with: self.jsonData(), options: []) as! NSDictionary
```

Comment:
This line going via NSDictionary caused `0` => `false` bug. 

Might have been fixed passing some options... but the overall solution with SwiftyJSON is better